app/js/messages_manager.js

```diff
@@ -3031,6 +3031,8 @@ angular.module('myApp.services')
               id: message.id,
               mid: mid
             })
+            clearDialogCache(mid)
+            reloadConversation(peerID)
```

Comment:
Unfortunately that is not a right way to do this update.
As you see `reloadConversation`  always performs an extra API request to get peer conversation. This is a bad idea, since every every edited message (even, when it's not a last in conversation) will result in API request, while we have already all the data about message changes in our `updateEditMessage` / `updateEditChannelMessage` update. The right way to do this is to create separate method, like `applyEditedTopMessage` where check if message is the top one for corresponding `peerID` (via `getDialogByPeerID(peerID)` method) and then broadcast `dialogs_multiupdate` event with required parameters.